cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala

```diff
@@ -25,199 +25,166 @@ import java.io.InputStream
 import com.rackspace.com.papi.components.checker.Config
 import com.rackspace.com.papi.components.checker.util.URLResolver
 import com.rackspace.com.papi.components.checker.wadl.WADLCheckerBuilder
-
-import org.clapper.argot.ArgotConverters._
-import org.clapper.argot.{ArgotParser, ArgotUsageException}
-
 import com.martiansoftware.nailgun.NGContext
+import scopt.OptionParser
 
 object Wadl2Checker {
   val title = getClass.getPackage.getImplementationTitle
-  val version = getClass.getPackage.getImplementationVersion
+  val implVersion = getClass.getPackage.getImplementationVersion
 
   def parseArgs(args : Array[String], base : String,
                 in : InputStream, out : PrintStream, err : PrintStream) : Option[(Source, Result, Option[StreamResult], Config)] = {
 
-    val parser = new ArgotParser("wadl2checker", preUsage=Some(s"$title v$version"))
+    val c = new Config
 
-    val input = parser.parameter[String]("wadl",
-                                         "WADL file/uri to read.",
-                                         false)
+    // Fix default values for fields with defaults that differ between the API and CLI Config
+    c.removeDups = false
 
-    val output = parser.parameter[String]("output",
-                                          "Output file. If not specified, stdout will be used.",
-                                          true)
+    var input: String = ""
+    var output: Option[String] = None
+    var outputMetadata: Boolean = false
 
-    val help = parser.flag[Boolean] (List("h", "help"),
-                                     "Display usage.")
+    val parser = new OptionParser[Unit]("wadl2checker") {
+      head(s"$title v$implVersion")
 
-    def source: Source = new StreamSource(URLResolver.toAbsoluteSystemId(input.value.get, base))
+      opt[Unit]('d', "remove-dups").text("Remove duplicate nodes. Default: false")
+        .foreach(_ => c.removeDups = true)
```

Comment:
Note that I did not include the `.optional()` designator since that is the default for `opt`s, but it could be included for explicitness, if desired.